src/atoms/headings/one/large/large-white/index.astro

```diff
@@ -0,0 +1,3 @@
+<h1 class="text-5xl md:text-8xl lg:text-9xl text-white font-medium font-merriweather">
+    <slot/>
+</h1>
```

Comment:
@Nandiniverma2024 rename the directory from headings/one/large/.. to headings/one/white/index.astro. Add a new line at the end of the file.